main.go

```diff
@@ -153,7 +153,6 @@ func mainError() error {
 	daemonCommand.PersistentFlags().Int(f.Service.Cluster.Kubernetes.IngressController.SecurePort, 0, "Secure port of guest cluster Ingress Controller.")
 	daemonCommand.PersistentFlags().String(f.Service.Cluster.Kubernetes.Kubectl.Docker.Image, "", "Full docker image of kubectl.")
 	daemonCommand.PersistentFlags().String(f.Service.Cluster.Kubernetes.Kubelet.AltNames, "", "Alternative names for guest cluster kubelet certificates.")
-	daemonCommand.PersistentFlags().String(f.Service.Cluster.Kubernetes.Kubelet.Labels, "", "Labels for guest cluster kubelets.")
```

Comment:
Instead of requiring a parameter to be passed for the kubelet labels, I added a function to the `key` package. This function mimics what `kubernetesd` is doing.